src/Components/MainApp/App.js

```diff
@@ -7,42 +7,35 @@ import ExperimentTwo from '../ExperimentTwo/ExperimentTwo.js';
 
 function App() {
   // Button States
-  const [headerState, setHeaderState]=useState(true);
-  const [experimentOneButtonState,setExperimentOneButtonState ]=useState(false);
-  const [experimentTwoButtonState,setExperimentTwoButtonState]=useState(false);
+  const[experimentButtonState,setExperimentButtonState]=useState('0')
+  const experimentButtons=[
+    {
+      name: 'Experiment 1',
+      id : '1'
+    },
+    {
+      name: 'Experiment 2',
+      id:'2'
+    }
+  ];
 
-  const handleExperimentOneEvent=()=>
-  {
-    setExperimentOneButtonState(true);
-    setExperimentTwoButtonState(false);
-    setHeaderState(false);
+  const handleExperimentButtonState = (experimentButtonSelected)=>{
+    setExperimentButtonState(experimentButtonSelected);
   }
-
-  const handleExperimentTwoEvent=()=>
-  {
-    setExperimentTwoButtonState(true);
-    setExperimentOneButtonState(false);
-    setHeaderState(false);
-  }
-
-  const backButtonState=()=>
-  {
-    setExperimentOneButtonState(false);
-    setExperimentTwoButtonState(false);
-    setHeaderState(true);
-  }
-
+  
   return (
       <div>
         <div>
-          {headerState && <Header/>}
-          {experimentOneButtonState && <ExperimentOne isBackButtonClicked={backButtonState}/>}
-          {experimentTwoButtonState && <ExperimentTwo />}
+          {(experimentButtonState === '0') && <Header/>}
+          {(experimentButtonState === '1') && <ExperimentOne isBackButtonClicked={()=>{setExperimentButtonState('0')}}/>}
```

Comment:
I don't think you need the braces around the function body when it's just a single statement so could do something like `isBackButtonClicked={() => setExperimentButtonState('0')}`

Reply:
Yup, agree. Will do that now